Vostok.Metrics.System/Host/DiskUsageCollector_Linux.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Vostok.Metrics.System.Helpers;
+
+namespace Vostok.Metrics.System.Host
+{
+    internal class DiskUsageCollector_Linux
+    {
+        private Dictionary<string, DiskStats> previousDisksStatsInfo = new Dictionary<string, DiskStats>();
+        private readonly Stopwatch stopwatch = new Stopwatch();
+
+        public void Collect(HostMetrics metrics, IEnumerable<string> diskstatsLines)
```

Comment:
Why doesn't this class read the file by itself? What's the point in delegating it upstream?